### PR TITLE
add change log page for pythom module

### PR DESCRIPTION
**Pull request by yongjuntian**

### CODE REVIEW

**Review by LouisHQ — APPROVED** [review]

File: doc/api/python/changelog/changelog.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- improve recording element experience on SAP, ensure the generated locator is SAP locator.
+
+## 0.1.0 (30 June 2022)
+Initial release version.
+- Several desktop automation libraries: UIA, Java, IA, SAP automation.
+- Several web automation libraries: Chrome, Edge, Internet Explorer(IE), FireFox.
+- Support automation based on image recognization.
+- Unified automation API for different applications, such as click, set_text, get_text, send_hotkey, select_item etc.
+- Special APIs for web automation: open browser, attach to already opened browser, get active tab, close browser, refresh tab, close tab etc.
+- Special api for application window: maximized, minimize, restore etc.
```
Comment (DiceyClickcorp) [inline doc/api/python/changelog/changelog.md:25]: maximize

File: doc/api/python/changelog/changelog.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Change Log
+## 0.1.1 (15 July 2022)
+### New features
+- New api: new_tab on browser class.
```
Comment (robinbob) [inline doc/api/python/changelog/changelog.md:4]: add reference for api?

File: doc/api/python/changelog/changelog.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- New api: clicknium.edge/chrome/firefox.extension.detect(), it can get browser extension installation state.
+- New api: get_screenshot, global interface to capture the screenshot.
+- Updated api: set_text with new parameter `append`, indicate whether append the text or overwrite the text on the target element.
```
Comment (robinbob) [inline doc/api/python/changelog/changelog.md:7]: change this parameter name to "setmode"? check this method: https://docs.microsoft.com/en-us/dotnet/api/system.io.filemode?view=net-6.0

File: doc/api/python/changelog/changelog.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+### Improvements
+- Fix yrate not working issue for api click, double_click.
+- Improve browser extension installation experience.
+- During running, if there are multiple windows matched, will choose the most active window defaultly.
+- Fix UIA bug: validation failed on windows file explorer's left tree.
+- Improve locator for web automation: trim the space for the value of attrbiute sinfo, name etc.
```
Comment (robinbob) [inline doc/api/python/changelog/changelog.md:14]: attribute

File: doc/api/python/changelog/changelog.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+### New features
+- New api: new_tab on browser class.
+- New api: clicknium.edge/chrome/firefox.extension.detect(), it can get browser extension installation state.
+- New api: get_screenshot, global interface to capture the screenshot.
+- Updated api: set_text with new parameter `append`, indicate whether append the text or overwrite the text on the target element.
+
+### Improvements
+- Fix yrate not working issue for api click, double_click.
+- Improve browser extension installation experience.
+- During running, if there are multiple windows matched, will choose the most active window defaultly.
```
Comment (robinbob) [inline doc/api/python/changelog/changelog.md:12]: The front active window will be chosed when there are multiple windows matched by given locator during automation run.

File: doc/api/python/changelog/changelog.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- Fix UIA bug: validation failed on windows file explorer's left tree.
```
Comment (robinbob) [inline doc/api/python/changelog/changelog.md:13]: on navigation pane of windows file explorer.

**Pull request merged**